src/workers/job-handler.js

```diff
@@ -56,21 +62,34 @@ async function createInteractionSteps(job) {
       answerOption = answerOptionStore[newId]['value']
     }
 
-    interactionSteps.push({
-      id: newId,
-      campaign_id: id,
-      question: step.question,
-      script: step.script,
-      answer_option: answerOption,
-      parent_interaction_id: parentId
-    })
+    dbInteractionStep = await InteractionStep
```

Comment:
Save fails for me after the first step unless this is `const dbInteractionStep = blah`